rskj-core/src/main/java/co/rsk/core/bc/BlockChainImpl.java

```diff
@@ -184,6 +184,14 @@ public long getSize() {
      */
     @Override
     public ImportResult tryToConnect(Block block) {
+        if (block == null)
+            return ImportResult.INVALID_BLOCK;
+
+        if (!block.isSealed()) {
+            panicProcessor.panic("unsealedblock", String.format("Unsealed block %s %s", block.getNumber(), Hex.toHexString(block.getHash())));
+            block.seal();
```

Comment:
If an unsealed block arrives here is because something out of the expected flow happened. I understand that it is why the panic is logged. 
What I don't get is why the block is sealed and accepted **instead** of returning INVALID_BLOCK?